stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -442,4 +442,21 @@ class GaussianDropout(Layer[tf.Tensor, tf.Tensor]):
         name: str | None = None,
     ) -> None: ...
 
+class MaxPooling2D(Layer[tf.Tensor, tf.Tensor]):
+    def __init__(
+        self,
+        pool_size: int | tuple[int, int] = (2, 2),
+        strides: int | tuple[int, int] | None = None,
+        padding: Literal["valid", "same"] = "valid",
```

Comment:
We might not be able to type this as a literal, the docs mention it's case-insensitive

https://www.tensorflow.org/api_docs/python/tf/keras/layers/MaxPool2D#args

there's examples of code that pass the value in all caps

https://github.com/search?q=padding%3D%22valid%22+language%3APython+MaxPool2D&type=code